_schemaV2.graphql

```diff
@@ -16942,6 +16945,14 @@ type SearchCriteriaLabel {
 # A search criteria or errors object
 union SearchCriteriaOrErrorsUnion = Errors | SearchCriteria
 
+enum SearchCriteriaSortEnum {
+  # Sort by most popular search criteria in descending order
+  COUNT_DESC
+
+  # Sort by high quality priority listed first
+  HIGH_QUALITY_DESC
+}
+
```

Comment:
There's a default sort on the gravity side of things if none are explicitly passed through